packages/typespec-client-generator-core/src/lib.ts

```diff
@@ -198,6 +198,18 @@ export const $lib = createTypeSpecLibrary({
         default: `Cannot flatten property of polymorphic type.`,
       },
     },
+    "conflict-access-override": {
+      severity: "warning",
+      messages: {
+        default: `@access override conflicts with the access calculated from operation or other @access override.`,
```

Comment:
i think it would be good to just add the name of the model that is conflicting

Reply:
the warning info is binding to the model type, and the diagnostics will help to show where the warning is come from, so i did not add name in the message.